subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+groups and mentors, one mentor may be responsible for two groups.  We suggest that students meet on
+Tues/Wed, with mentor on Friday or weekend, if convenient for the group. 
+
+### Mini-lessons
+
+The following is the proposed mini-lesson plan:  
+
+1. Trainees will in collaboration with their mentor figure out a portion of material that the student
+will teach from. The aim is to have approximately an hours worth of material. The trainee will then
+prepare the material, including data, and make it available for review by the mentor. 
+2. The mentor will then review the lesson plan and have a short (online) meeting with each trainee one
+week before the trainee is scheduled to teach their Mini-lesson. Guidance on what to teach and how to
+teach it will be given both via email and during face-to-face communication. 
+3. The trainee teaches the 1 hr Mini-lesson, in person, to a minimum of 1 other individual while the
+mentor observes either in person, or remotely. 
+4. Directly after the Mini-lesson, the trainee and mentor engage in a debrief where the mentor first
+asks the trainee to comment on their performance before they give feedback on the trainee’s performance. 
+5. Within a week or two after the trainee has completed teaching their Mini-lesson, the mentor and
+trainee will meet together for a debrief. The trainee will before this session fill out a short
+questionnaire about how their lesson went, which will feed into the discussion with their mentor.
+Questionnaire will be provided by the SCF. This debrief will preferably happen in groups, as the mentor
+is likely to have several trainees that have taught the same material for their Mini-lessons. 
+6. Trainees will be encouraged to make a suggested revision/edit/addition (e.g. new challenge question)
+to the lesson topic they taught.  
+
+## Timeline
+
+### Preparation
+
+- Week 1: signup forms for mentors and trainees available, announced with blogpost, as well as email to
+discuss list and all recently educated instructors. 2 week signup period. 
+- Week 3: selection of mentors, creation of groups, combine mentors and groups. Send out info end of
+Week 3. 
+- Week 4: Orientation meeting for trainees, orientation meeting and training for mentors.  
+
+### Workthrough Phase
+
+Weeks 5-9: 4 weeks where each group goes through a part of the core curriculum described below for 
+either DWC or DC.
+
+### Mini Lessons
+
+- Week 9: break 
+- Week 10: student puts together mini-lesson material 
+- Week 11: mentor reviews material 
+- Week 12: mentor observes mini-lessons 
+
+*Core Material
+
+For SWC: shell, git and either R or Python
+
+For DC: Spreadsheets, data cleaning with OpenRefine, Sql, and either R- or Python-ecology
+
+* * *
+
+Action items: 
+
+- Create mentor “ad” blog post with clear expectations of how much time is involved, what training will
+be provided, and what the mentor will need to do 
```

Comment:
and what the mentors will get in return... again, without clear and attractive incentives, I don't think many people will sign up for this significant time commitment.